CPAC/nuisance/nuisance.py

```diff
@@ -2450,9 +2450,16 @@ def nuisance_regression(wf, cfg, strat_pool, pipe_num, opt, space):
             opt = regressor_dct
             break
 
+    bandpass = 'Bandpass' in opt
+    bandpass_before = bandpass and cfg['nuisance_corrections',
+                                       '2-nuisance_regression',
+                                       'bandpass_filtering_order'] == 'Before'
+
     nuis = create_nuisance_regression_workflow(opt, name='nuisance_regression'
                                                f'_{space}_{opt["Name"]}'
                                                f'_{pipe_num}')
+    if bandpass_before:
+        nofilter_nuis = nuis.clone(name=f'{nuis.name}-noFilter')
```

Comment:
Is [this](https://github.com/FCP-INDI/C-PAC/pull/1824#figure) a reasonable thing to do? Ie, is it reasonable to run nuisance twice in the bandpass-before case to get a non-filtered bold image?